frameworks/Crystal/moonshine/setup-postgres.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-fw_depends crystal
+curl http://dist.crystal-lang.org/apt/setup.sh | sudo bash
+
+sudo apt-get install crystal
```

Comment:
These lines are 100% wrong - use the `fw_depends` structure in all setup files. If these are the correct steps (and I am not saying that they are), then they should be in [this file](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/languages/crystal.sh), not in setup files like this one.